### DISCUSSION TITLE
display=column doesn't work properly

### DISCUSSION THREAD

**Discussion by Erik-Egnatosyan** [lowlighter/metrics#1404] [Q&A]
When I add display=column in GitHub README page, Elements don't fit in columns, it just goes vertically.

**Comment by lowlighter** [comment]
Hi 👋 !

It may be because GitHub redesigned a bit the profile page so the `@media-queries` doesn't match anymore the correct sizing.
However as #839 mentions, there are some known issue about `config_display: columns` so I would advise against using it and use `config_display: large` instead for now
